website/static/js/registrationUtils.js

```diff
@@ -463,38 +471,45 @@ RegistrationEditor.prototype.create = function(schemaData) {
 };
 RegistrationEditor.prototype.submit = function() {
     var self = this;
-    
+
     var currentNode = window.contextVars.node
     var currentUser = window.contextVars.currentUser
     var url = '/api/v1/project/' + currentNode.id +  '/draft/submit/' + currentUser.id + '/'
```

Comment:
Please use ```self.urls.submit```. The Editor should not construct URLs internally.

Also, this route does not need the current user's id, that is always available server-side.  So let's remove <uid> from the route.

In the view ```user = auth.user```